setup.py

```diff
@@ -54,6 +54,7 @@
         "Flask-Celery-Helper",
         "Flask-Migrate",
         "Flask-OAuthlib",
+        "Flask-Recaptcha",
```

Comment:
I think you'll need to add it to `requirements.txt` too; `pip freeze` should tell you the currently installed version...